packages/framework/src/Resources/config/services.yaml

```diff
@@ -11,108 +11,74 @@ parameters:
 
 services:
     _defaults:
-        autowire: true
         autoconfigure: true
+        autowire: true
         public: false
 
     Shopsys\FrameworkBundle\:
-        resource: '../../**/*{Calculation,Dispatcher,Facade,Factory,Formatter,Generator,Handler,InlineEdit,Listener,Loader,Mapper,Middleware,Parser,Provider,Recalculator,Registry,Repository,Resolver,Service,Scheduler,Subscriber,Transformer,DataFetcher}.php'
         exclude: '../../{Command,Controller,DependencyInjection,Form,Migrations,Resources,Twig}'
-
-    Shopsys\FrameworkBundle\Model\AdvancedSearch\Filter\:
-        resource: '../../Model/AdvancedSearch/Filter/*Filter.php'
-
-    Shopsys\FrameworkBundle\Model\AdvancedSearchOrder\Filter\:
-        resource: '../../Model/AdvancedSearchOrder/Filter/*Filter.php'
+        resource: '../../**/*{Calculation,Dispatcher,Facade,Factory,Formatter,Generator,Handler,InlineEdit,Listener,Loader,Mapper,Middleware,Parser,Provider,Recalculator,Registry,Repository,Resolver,Service,Scheduler,Subscriber,Transformer,DataFetcher}.php'
 
     Shopsys\FrameworkBundle\Controller\:
         resource: '../../Controller/'
         tags: ['controller.service_arguments']
 
-    doctrine.orm.default_entity_manager:
```

Comment:
It's quite weird for me to have entries with the lower first case at the end. I prefer to ignore cases for sorting or having lowercase entries first... But I can live with this solution

Reply:
I cannot configure this sadly